src/main/java/com/jcraft/jsch/KnownHosts.java

```diff
@@ -290,7 +295,7 @@ public int check(String host, byte[] key){
   public void add(HostKey hostkey, UserInfo userinfo){
     int type=hostkey.type;
     String host=hostkey.getHost();
-    byte[] key=hostkey.key;
+//    byte[] key=hostkey.key;
```

Comment:
If this isn't used, then I would say to remove it instead of commenting it out.

Reply:
`key` is used in a code block that currently is commented out a couple of lines later. So the person who commented out that block simply forgot to do it with this particular line. That's why I'd keep it there until it's decided what should happen with the larger code block. If it's "reactivated", the line can be brought back to life as well and if it's removed that line has to go, too.

Reply:
Ok, you are correct on this one: it is best to just comment it out for now as you have done.